markdown/introduction/04-callbacks_and_events.md

```diff
@@ -671,35 +682,16 @@ saved_values.saveval
 
 Go back to the Harmonic oscillator. Use the `SavingCallback` to save an array for the energy over time, and do this both with and without the `ManifoldProjection`. Plot the results to see the difference the projection makes.
 
+````
+Error: ArgumentError: Package DiffEqTutorials not found in current path:
```

Comment:
Ah, DiffEqTutorials is not in the inner Project.toml. How do we make the environments stack?